### PR TITLE
Feature/fry update confirmation pages

### PR DESCRIPTION
**Pull request by FernandoBoza**
## Description
Deconstructs the Confirmation page to use the ConfirmationResults component to handle the logic and UI for the different Confirmation pages creation, specifically the Under Review condition.

## Testing done
- [ ] Done

### CODE REVIEW

File: src/applications/fry-dea/components/ConfirmationResponses.jsx [hunk 1]
```diff
@@ -83,35 +83,36 @@ export function UnderReview(response, name) {
...
         <ul>
-          {/* ▪	Learn more about VA benefits and programs through the Building Your Future with the GI Bill Series. */}
-          {/* ▪	Measure your interests and skill levels and help figure out your career path with CareerScope®. */}
-
           <li>
-            Review and/or update your direct deposit information on your{' '}
-            <a href="/profile">VA.gov profile</a> .
+            If you need to submit documentation to VA, such as service records,
+            please send this through our
+            <a href="https://nam04.safelinks.protection.outlook.com/?url=https%3A%2F%2Fask.va.gov%2F&data=04%7C01%7Cherbert.anagho%40accenturefederal.com%7C5b0be35e33a2487d4a0c08d9ecb991bc%7C0ee6c63b4eab4748b74ad1dc22fc1a24%7C0%7C0%7C637801104030719343%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000&sdata=QuGxWs9osAHjaGwInFjQO5cwEQ%2BK84u9J3XH2QcwZNk%3D&reserved=0">
```
Comment (nathanielbuck) [inline src/applications/fry-dea/components/ConfirmationResponses.jsx:89]: Can you double check this URL? It looks like an Outlook version of a link going through some sort of phishing/scam protection.

**Review by nathanielbuck — CHANGES_REQUESTED** [review]

**Review by nathanielbuck — APPROVED** [review]

File: src/applications/fry-dea/components/ConfirmationResponses.jsx (hunk 1 above)
Comment (FernandoBoza) [reply src/applications/fry-dea/components/ConfirmationResponses.jsx]: Updated Ask VA Feature link to https://ask.va.gov/

File: src/applications/fry-dea/components/ConfirmationResponses.jsx [hunk 2]
```diff
@@ -31,25 +31,25 @@ export function UnderReview(response, name) {
...
-                  <span>{format(new Date(), 'MMM d, yyyy')}</span>
-                </li>
-              </ul>
+          {/* {response ? ( */}
+          {/*  <> */}
+          <ul className="claim-list">
+            <li>
+              <strong>Date received</strong>
+              <br />
+              <span>{format(new Date(), 'MMM d, yyyy')}</span>
```
Comment (nathanielbuck) [inline src/applications/fry-dea/components/ConfirmationResponses.jsx:40]: A definition list would be a little more semantic here.

https://github.com/department-of-veterans-affairs/vets-website/blob/main/src/applications/my-education-benefits/components/confirmation/ConfirmationApproved.jsx#L39

Comment (FernandoBoza) [reply src/applications/fry-dea/components/ConfirmationResponses.jsx]: Updated HTML tags for the Date received from UL to DL.

File: src/applications/fry-dea/sass/fry-dea.scss [hunk 3]
```diff
@@ -129,7 +129,7 @@ $error-left-padding-width: 1.5rem;
 // END - Benefit Selection Page
 
 // Confirmation Page: Under Review
-.meb-hr#custom-fry-hr {
+#custom-fry-hr {
```
Comment (nathanielbuck) [inline src/applications/fry-dea/sass/fry-dea.scss:132]: In general, we should always use classes and only use IDs with rare exception. Two reasons:
- using an ID prevents us from using this element/applying this style twice on the same page;
- if we need to override any part of this styling, we can't do it with a class.

Comment (FernandoBoza) [reply src/applications/fry-dea/sass/fry-dea.scss]: Reverted to class based styling.

**Pull request merged**